README.md

```diff
@@ -179,12 +265,1268 @@ if($player instanceof Player && $player->isOnline()){
 }
 ```
 
+- registerCustom Preset
+
+First, build a custom preset registry
+CustomCameraPresetRegistry.php
```

Comment:
This should be provided by the virion

Reply:
This will reduce the code load by actually implementing the abstract class